src/Connectors/test/Connectors.Test/SqlServer/SystemData/SqlServerConnectorTest.cs

```diff
@@ -13,6 +13,8 @@
 using Steeltoe.Connectors.SqlServer;
 using Steeltoe.Connectors.SqlServer.RuntimeTypeAccess;
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Do we need a follow-up to deal with whatever is now obsolete?

Reply:
This is related to the bullet about `X502Certificate2` in the description of #1368. I haven't investigated further because we're not targeting .NET 9 in libraries. There's valuable information in the linked Aspire thread, but I'm not sufficiently familiar with the various kinds of certificates to decide on how to proceed.